coa/pkg/apis/v1alpha2/host/host.go

```diff
@@ -58,10 +65,11 @@ type VendorSpec struct {
 	LoopInterval int
 }
 type APIHost struct {
-	Vendors              []VendorSpec
-	Bindings             []bindings.IBinding
-	SharedPubSubProvider pv.IProvider
-	ShutdownGracePeriod  time.Duration
+	Vendors               []VendorSpec
+	Bindings              []bindings.IBinding
+	SharedPubSubProvider  pv.IProvider
+	SharedKeyLockProvider pv.IProvider
```

Comment:
Do you need to maintain a sharedKeyLockProvider in the APIHost? Looks like you are not passing the provider to manager init function as pubsub provider.

Reply:
Good point, looks like we don't need to keep sharedKeyLockProvider in the APIHost. And Global mode can be replaced by Shared mode. I will delete the Global mode

Reply:
I just found I cannot delete the Global mode. since Global mode is used for the config initialization

Reply:
I think we can keep the sharedKeyLockProvider in the APIHost, since this can be tracked if we have initialized the global lock provider before.